package/src/prmon.cpp

```diff
@@ -331,6 +334,10 @@ int main(int argc, char* argv[]) {
         << "                          all monitors enabled by default\n"
         << "                          Special name '[~]all' sets default "
            "state\n"
+        << "[--level, -l lev]         Set the minimum logging level of all "
+           "monitors\n"
+        << "[--level, -l mon:lev]     Set the minimum logging level of a "
+           "specific monitor\n"
```

Comment:
Can you give the valid values for the logging levels here please - I tried to use `DEBUG` and it didn't work 🤔

Reply:
I think this might be case sensitive, `-l cpumon:debug` seems to work but not `-l cpumon:DEBUG`.

Reply:
You are right, @amete. Do you think it is better to make it case insensitive?

Reply:
Yes, I think that would make life easier on the user side, would you agree @graeme-a-stewart?

Reply:
Yes, converting the string to the right case internally would help